src/Native/Unix/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1095,7 +1205,7 @@ int32_t CryptoNative_BioTell(BIO* bio)
         return -1;
     }
 
-    return BIO_tell(bio);
+    return (int32_t)BIO_ctrl(bio, BIO_C_FILE_TELL, 0, NULL);
 }
```

Comment:
BIO_tell/BIO_seek is not available on BoringSSL, but the underlying BIO_ctrl is. I can keep it this way or add it into the opensslshim.h as a macro.